packages/flutter_tools/lib/src/android/android_workflow.dart

```diff
@@ -288,6 +292,7 @@ class AndroidLicenseValidator extends DoctorValidator {
 
   Future<bool> _checkJavaVersionNoOutput() async {
     final String? javaBinary = Java.find(
+      config: _config,
```

Comment:
do we need to call this function again if we already have a `final Java? _java;` in scope?

Reply:
Good catch. Removed with commit [remove extraneous call to Java.find](https://github.com/flutter/flutter/pull/128264/commits/8300ebbd7c6a5b8880d32a4ec0f5dacf66b2578d)